README.md

```diff
@@ -255,108 +227,121 @@ services:
             - { name: pagerfanta.view }
 ```
 
-Reusing Options
----------------
+## Retrieving Views
 
-Sometimes you want to reuse options of a view in your project, and you don't
-want to write them all the times you render a view, or you can have different
-configurations for a view and you want to save them in a place to be able to
-change them easily.
+You can access the Pagerfanta views through the `pagerfanta.view_factory` service, which is a `Pagerfanta\View\ViewFactoryInterface` instance. This is useful if your application does not use Twig but you still want to use Pagerfanta views for rendering pagination lists.
 
-For this you have to define views with the special *OptionableView*:
+```php
+<?php
+
+namespace App\Service;
+
+use Pagerfanta\Pagerfanta;
+use Pagerfanta\View\ViewFactoryInterface;
+
+final class PagerfantaService
+{
+    private $viewFactory;
+
+    public function __construct(ViewFactoryInterface $viewFactory)
+    {
+        $this->viewFactory = $viewFactory;
+    }
+
+    public function render(Pagerfanta $pagerfanta, string $view, array $options = []): string
+    {
+        return $this->viewFactory->get($view)->render($pagerfanta, $this->createRouteGenerator($options), $options);
+    }
+}
+```
+
+## Reusing Options
+
+Sometimes you want to reuse options for a view in your project and you don't want to repeat those options each time you render a view, or you have different configurations for a view and you want to save those configurations to be able to change them easily.
+
+For this you can define views with the `Pagerfanta\View\OptionableView` class, which is a decorator for any `Pagerfanta\View\ViewInterface` instance.
```

Comment:
Not right now, the bundle doesn't expose a notion of "these are global options applied to all `{{ pagerfanta() }}` uses.  Even if that existed, I'd still leave this doc section in place since it's a good guide for setting up more specific reusable configurations for different occasions (say you've got an application that has different frontend and backend themes, you might want to default the backend pager to a different set of options from the frontend pager).

Reply:
What I meant is, what is the difference between using OptionableView and this:
```yaml
    pagerfanta.view.twig:
        class: App\Pagerfanta\View\TwigView
        arguments:
            - '@twig'
            - '@@BabDevPagerfantaBundle/default.html.twig'
        public: false
        tags: [{ name: pagerfanta.view, alias: twig }]

    pagerfanta.view.shop:
        class: App\Pagerfanta\View\TwigView
        arguments:
            - '@twig'
            - '@@BabDevPagerfantaBundle/shop.html.twig'
        public: false
        tags: [{ name: pagerfanta.view, alias: shop }]
```
Oh wait, this is about options other than the template, right?

Reply:
Right.  A default template can be set through the constructors on the Twig view (same way the `default_view` parameter is pushed into the Twig runtime class constructor to allow not explicitly specifying a view every time you call `{{ pagerfanta(pager) }}`), but not a default `$options` array.  And TBH, I'd just use the `OptionableView` decorator for that instead of trying to duplicate its logic.